udata_front/theme/gouvfr/templates/suivi.html

```diff
@@ -17,42 +17,165 @@
+                <li><strong>Données relatives à la newsletter</strong> : adresse e-mail.</li>
+            </ul>
+            
+            <h2>Pourquoi traitons-nous des données à caractère personnel ?</h2>
+            
+            <p class="text-justify">
+                <a href="http://data.gouv.fr">data.gouv.fr</a> est la plateforme nationale ouverte et communautaire qui vise à centraliser et structurer les données ouvertes présentant un intérêt public.
+                Elle s’adresse à ceux qui produisent des données et qui souhaitent ouvrir leurs données, à ceux qui exploitent des données et qui veulent télécharger des données et partager leurs réalisations, 
+                à toute personne qui souhaite découvrir des données ou trouver une information. 
+                La plateforme traite des données identifiantes pour permettre à des Contributeurs de publier des jeux de données et des réutilisations.
+            </p>
 
+            <h2>Qu’est-ce qui nous autorise à traiter des données à caractère personnel ?</h2>
+            
             <p class="text-justify">
-                Ce site dépose un petit fichier texte (un « cookie ») sur votre ordinateur lorsque vous le
-                consultez.
-                Cela nous permet de mesurer le nombre de visites et de comprendre quelles sont les pages les plus
-                consultées.
+                <a href="http://data.gouv.fr">data.gouv.fr</a> traite des données à caractère personnel en se basant sur : 
             </p>
+            <ul>
+                <li>L’exécution d’une mission d’intérêt public dont est investi le responsable de traitement au sens de l’article 6-1 e) du RGPD.</li>
+            </ul>
+            <p class="text-justify">
+                Cette mission d’intérêt public se traduit en pratique par : 
+            </p>
+            <ul>
+                <li>Le <a href="https://www.legifrance.gouv.fr/loda/id/JORFTEXT000039281619">décret n° 2019-1088 du 25 octobre 2019</a> relatif au système d’information et de communication de l’Etat et à la direction interministérielle du numérique, notamment son article 6 ;</li>
+                <li>La <a href="https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000024072788">circulaire du 26 mai 2011</a> relative à la création du portail unique des informations publiques de l’État “data.gouv.fr” ;</li>
+                <li>La <a href="https://www.legifrance.gouv.fr/circulaire/id/45162">circulaire du 27 avril 2021</a> relative à la politique publique de la donnée, des algorithmes et des codes sources.</li>
+            </ul>
 
-            <iframe class="well well-grey-100 fr-my-2w w-100" height="150"
-                src="https://stats.data.gouv.fr/index.php?module=CoreAdminHome&action=optOut&language=fr"></iframe>
+            <h2>Pendant combien de temps conservons-nous ces données ?</h2>
+            
+            <table>
```

Comment:
Can you wrap the tables in a `<div class='fr-table'>`  to have the DSFR styling applied ?